en/docs/use-cases/streaming-tutorials/creating-etl-application-via-tooling.md

```diff
@@ -45,7 +45,7 @@ In this tutorial, let's create the same Siddhi application created in [Performin
             `CREATE TABLE SweetProductionTable (name VARCHAR(20),amount double(10,2));`<br/> 
     - Download `productions.csv` file from [here](https://github.com/wso2/docs-ei/tree/master/en/streaming-integrator/docs/examples/resources/productions.csv) and save it in a location of your choice. (e.g., in `/Users/foo`).<br/>    
     - Download and install [Streaming Integrator Tooling](https://wso2.com/integration/streaming-integrator/#)<br/>          
-    - Download and install the [siddhi-io-cdc](https://siddhi-io.github.io/siddhi-io-cdc/) extension. For instructions, see [Downloading and Instaling Siddhi Connectors](../connectors/downloading-and-Installing-Siddhi-Extensions/).
+    - Download and install the [siddhi-io-cdc](https://siddhi-io.github.io/siddhi-io-cdc/) extension. For instructions, see [Downloading and Instaling Siddhi Connectors]({{base_path}}/streaming/connectors/downloading-and-installing-siddhi-extensions.md).
```

Comment:
Dont add ".md" at the end of the link. If I recall correctly, the link breaks when .md is added.